public/framework/catalog_cache/cache_manager.test.tsx

```diff
@@ -125,7 +125,7 @@ describe('CatalogCacheManager', () => {
     });
 
     it('should return default cache object if cache is not found', () => {
-      const defaultCacheObject = { version: CATALOG_CACHE_VERSION, dataSources: [] };
+      const defaultCacheObject = { version: CATALOG_CACHE_VERSION, dataSources: [],'client_id':'' };
```

Comment:
As we are planning to make `clientId` optional, we shouldn't change the older tests and add new tests which dummy `clientId` value. This will really help us in future to catch regressions.